options/posix/generic/netdb-stubs.cpp

```diff
@@ -7,6 +7,13 @@
 #include <stdlib.h>
 #include <stddef.h>
 
+#undef h_errno
+int h_errno;
```

Comment:
Shouldn't this be `__thread_local` like the normal errno?

Reply:
It should probably be called `__mlibc_h_errno` since it's a "anonymous" global variable.